src/ripple/websocket/Connection.h

```diff
@@ -262,8 +262,7 @@ Json::Value ConnectionImpl <WebSocket>::invokeCommand (Json::Value& jvRequest)
     Json::Value jvResult (Json::objectValue);
 
     auto required = RPC::roleRequired (jvRequest[jss::command].asString());
-    Role const role = requestRole (required, port_, jvRequest, m_remoteAddress,
-                                   getConfig().RPC_ADMIN_ALLOW);
+    Role const role = requestRole (required, port_, jvRequest, m_remoteAddress);
```

Comment:
Feel like renaming `port_` to `m_port` while you're in there? (I know it's not new. It's private and only used a couple place and it's good hygiene.)